book/website/communication/personal.md

```diff
@@ -0,0 +1,42 @@
+(cm-personal)=
+# Communication of Personal Skills and Research
+
+(cm-personal-prerequisites)=
+## Prerequisites
+
+| Prerequisite | Importance | Skill Level | Notes |
+| -------------|----------|------|----|
+| [](#rr-open) | Helpful | Beginner | An understanding of open research practices is useful for creating a personal website. |
+| [](#cm-blogs) | Helpful | Beginner | Blogs for research communication can be part of a personal website. |
+| [](#cm-citable) | Helpful | Beginner | Gain understanding of how to make your research objects citable. |
+
+(cm-personal-summary)=
+## Summary
+
+This chapter explores the importance and benefits of communicating your skills and your research via various pathways, including a personal website and a narrative or evidenced-based CV.
+
+The section on personal websites provides a general structure with common components and content suggestions for developing personal websites.
+Then, coding and non-coding website generators with different levels of complexity are discussed, explaining their features and differences.
+Several deployment options and maintenance strategies are also presented, helping researchers to choose the best solution for their needs.
+
+The section on narrative or evidenced-based CVs explains what these CVs are, why they are currently used in research evaluation, and how you can start with writing one. 
+The section contains examples and tips on how to write a narritive CV, and contains an overview of what type of research objects you can include in the CV. 
+
+Both sections conclude with a list of additional resources.
+
+(cm-personal-motivation)=
+## Motivation and Background
+
+It is important to advertise your research skills and output via various pathways.
+Examples are a personal website or a narrative CV. 
+
+A personal website is an online space where researchers can share their work and additional interests with a broad audience.
+It can serve as a platform to present research findings, projects, talks, publications, CV or resume, and other professional information.
+But this space is not necessarily limited to academic content; it can also be a place to share personal interests, hobbies, and other activities.
+As a result, this website can be customised to express the individual's unique identity and values.
+This online presence can help researchers expand their visibility, engage with a wider audience, connect with potential collaborators, and even create new career opportunities.
+
+The narrative CV aims to reward a broader set of research-related activities beyond peer-reviewed publications, valuing diverse contributions and career paths in research. 
+This will lead to more inclusion and diversity in research, as well as encouraging a more responsible manner of using metrics.
+It also allows researchers to place focus on activities that they value, instead of being limited by a restricted amount of criteria.
+
```

Comment:
```suggestion
It also allows researchers to place focus on activities that they value, instead of being limited by a restricted amount of criteria.
```

Removing a blank line at the end of the file.